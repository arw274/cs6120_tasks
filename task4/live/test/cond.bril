@main (input: int) {
    one: int = const 1;
    cond0: bool = ge input one;
    br cond0 .ge .zero;
    b: int = const one;
.ge:
    input: int = sub input one;
.zero:
    ret;
}