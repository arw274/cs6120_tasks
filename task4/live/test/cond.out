Function: main

Block 0 live variables at input:
['input']
Block 1 live variables at input:
['input', 'one']
Block ge live variables at input:
['input', 'one']
Block zero live variables at input:
[]

